source/jormungandr/jormungandr/street_network/geovelo.py

```diff
@@ -88,6 +92,24 @@ def __init__(
         if not is_url(service_url):
             raise ValueError('service_url {} is not a valid url'.format(service_url))
         self.service_url = service_url
+
+        # If shape is absent, it should work as a normal street_network connector as before for retro-compatibility
+        if zone:
+            self.polygon_zone = Polygon(zone)
```

Comment:
use catch around `Polygon` and log the exception

Reply:
:+1: